app/templates/app/create_list.html

```diff
@@ -0,0 +1,11 @@
+{% extends 'app/base.html' %}
+{% block content %}
+    <h2>Create List</h2>
+    <form method="post" novalidate>
+        {% csrf_token %}
+        {% for list in list_view %}
```

Comment:
make sure to named it correctly, doesn't make sense list in list_view

Reply:
Done sir